creating_images/guidelines.adoc

```diff
@@ -273,9 +273,23 @@ that a process which is able to escape the container due to a vulnerability in
 the container framework will not have specific user permissions on the host
 system.
 
-To accomplish this, directories that must be written to by processes in the
-image should be world-writable. In addition, the processes running in the
-container must not listen on privileged ports (ports below 1024).
+To support running as an arbitray user, directories and files that must be written to by 
```

Comment:
@danmcp those examples are for persistent storage permissions, not permissions of files within the image, pretty sure it's unrelated.

but i'll update this to make it clearer it's referring to how you define files within the image during image construction.